docs/mypy.md

```diff
@@ -76,3 +76,59 @@ pass the `--all` option to tools/run-mypy.
 
 If you type annotate some of those files, please remove them from the
 exclude list.
+
+## Mypy cheat sheet
+
+Refer to the [mypy cheat sheet](https://github.com/python/mypy/blob/master/docs/source/cheat_sheet.rst)
+to quickly get started with annotating code.
```

Comment:
I'd move this up basically in the first section -- it's a better link than either of the two we have up there for folks either annotating new code or trying to figure out why their code is wrong...